src/services/mentees.js

```diff
@@ -1042,6 +1056,18 @@ module.exports = class MenteesHelper {
 				additionalProjectionString,
 				false
 			)
+			mentorExtensionDetails = await mentorQueries.getMentorsByUserIdsFromView(
```

Comment:
@nevil-mathew Why was this added multiple times?

Reply:
That is for the pagination logic. This is the existing flow. @rakeshSgr